Sources/Sentry/PrivateSentrySDKOnly.m

```diff
@@ -12,7 +12,7 @@
 
 @implementation PrivateSentrySDKOnly
 
-static SentryOnAppStartMeasurementAvailable _onAppStartMeasurmentAvailable;
+static SentryOnAppStartMeasurementAvailable _onAppStartMeasurementAvailable;
```

Comment:
Again 🥇, but https://github.com/getsentry/sentry-cocoa/pull/2154 would be better IMO.

Reply:
![good-fine](https://user-images.githubusercontent.com/3241469/190246256-1ff40e3d-13b2-4f30-b8cc-3b75480fc3d6.gif)